### DISCUSSION TITLE
Unsuccessful with "OptiCore + LuxCore Lens" tutorial

### DISCUSSION THREAD

**Discussion by Garrett-Lundegard** [CodeFHD/OptiCore#10] [Q&A] [answered]
[TripleLens005.zip](https://github.com/CodeFHD/OptiCore/files/6748286/TripleLens005.zip)
I'm trying to recreate simulation done in this tutorial: https://www.youtube.com/watch?v=3hln88ukiZI&t=615s

Here is the result I am getting:
![image](https://user-images.githubusercontent.com/33532122/124028867-7c772d00-d9c2-11eb-97ee-84f73613fd44.png)

It's a pretty picture, but not quite the symmetrical pattern achieved in the tutorial. Any ideas as to what I may have done wrong? 
I remade each lens to ensure there was no error there, but no luck.

**Comment by Garrett-Lundegard** [comment, marked as answer]
By changing setting N1 and N2  = 64 during lens creation, I managed to get this picture:
![image](https://user-images.githubusercontent.com/33532122/124152907-eb579300-da61-11eb-91d4-feeff40bb262.png)

Looks much better, but I wonder what is the difference that results in this image versus the patterns shown in the image from the tutorial.

**Reply by CodeFHD** [reply]
I have noticed this kind of behaviour also, but I am unsure of the reason behind it. If you increase N1 or N2 in small steps, there is a sudden jump at some point. I also did not do extensive tests if this depends on other parameters like lens curvature.
